ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+/**
+ * @module Page::MessagesList
+ * Page::MessagesList components are used to display breadcrumb links. This is component will be replaced when HDS system is incorporated
```

Comment:
Is this description accurate. Sounds like maybe a copy / paste from the breadcrumbs component.